rfc6962-bis.xml

```diff
@@ -766,7 +776,7 @@ messages.
           <t hangText="error_code:">
             An error code readable by the client. Some codes are generic and are detailed here. Others are detailed in the individual requests. Error codes are fixed text strings.
             <list style="hanging">
-              <t hangText="not compliant">
+              <t hangText="'not compliant'">
```

Comment:
How about putting the error strings and descriptions into a 2-column <texttable> ?